src/test/bufferlist.cc

```diff
@@ -808,7 +808,7 @@ static void bench_bufferlistiter_deref(const size_t step,
   }
   utime_t end = ceph_clock_now();
   cout << bufsize * bufnum << " derefs over bl with " << bufnum
-       << " buffers, each " << bufsize << " bytes long"
+       << " buffers, each " << buf.size() << " bytes long"
```

Comment:
why?

Reply:
It's better to use buf.size() instead of bufsize because "bl.append(ceph::bufferptr(buf.c_str(), buf.size()));"

Reply:
could you update the commit message to explain this? something like

> use buf.size() instead of bufsize, because `buf` is used to stuff `bl` in a batch of `buf.size()`.

but i don't think this helps with the readability or make the code more correct, as the parameters of this function explicitly denote the `bufsize` and `bufnum`.

Reply:
Thanks Kefu, I'll keep the original code there since 1) std::string is from standard library  2) this PR doesn't improve readability.

Reply:
thanks. yeah, we rely on the behavior of `std::string` here.